pyracing/response_objects/historical_data.py

```diff
@@ -94,84 +94,84 @@ def __init__(self, dict):
-        self.starts = dict['15']
-        self.sublevel = dict['35']
-        self.top25pcnt = dict['1']
+        self.row = dict['37']
+        self.starts_total = dict['15']
+        self.sub_level = dict['35']
+        self.top_25_percent = dict['1']
         self.ttrating = dict['25']
         self.ttrating_rank = dict['6']
-        self.wins = dict['18']
+        self.win_total = dict['18']
         # These are only used in front end webpage; Not in dict results
         # self.custrow = dict['17']
         # self.rowcount = dict['20']
 
 
 class WorldRecords:
     def __init__(self, dict):
-        self.carid = dict['9']
+        self.car_id = dict['9']
         self.category = parse_encode(dict['35'])
-        self.catid = dict['10']
-        self.clubid = dict['7']
-        self.clubname = parse_encode(dict['27'])
-        self.countrycode = parse_encode(dict['6'])
-        self.custid = dict['31']
-        self.displayname = parse_encode(dict['29'])
-        self.helmcolor1 = parse_encode(dict['28'])
-        self.helmcolor2 = parse_encode(dict['30'])
-        self.helmcolor3 = parse_encode(dict['26'])
-        self.helmpattern = dict['22']
+        self.cat_id = dict['10']
+        self.club_id = dict['7']
+        self.club_name = parse_encode(dict['27'])
+        self.country_code = parse_encode(dict['6'])
+        self.cust_id = dict['31']
+        self.display_name = parse_encode(dict['29'])
+        self.helm_color1 = parse_encode(dict['28'])
+        self.helm_color2 = parse_encode(dict['30'])
+        self.helm_color3 = parse_encode(dict['26'])
+        self.helm_pattern = dict['22']
         self.irating = dict['4']
-        self.licenseclass = parse_encode(dict['3'])
-        self.licensegroup = dict['14']
-        self.licenselevel = dict['23']
+        self.license_class = parse_encode(dict['3'])
+        self.license_class = dict['14']
+        self.license_level = dict['23']
         self.practice = parse_encode(dict['2'])
         self.practice_start_time = dict['8']
-        self.practice_subsessionid = dict['13']
+        self.practice_subsession_id = dict['13']
         self.qualify = parse_encode(dict['15'])
         self.qualify_start_time = dict['21']
-        self.qualify_subsessionid = ['36']
+        self.qualify_subsession_id = ['36']
         self.race = parse_encode(dict['19'])
         self.race_start_time = dict['18']
-        self.race_subsessionid = ['11']
+        self.race_subsession_id = ['11']
         self.region = parse_encode(dict['34'])
-        self.rn = dict['33']
+        self.row = dict['33']
         self.season_quarter = parse_encode(dict['12'])
         self.season_year = parse_encode(dict['17'])
-        self.sublevel = dict['32']
+        self.sub_level = dict['32']
         self.timetrial = parse_encode(dict['37'])
         self.timetrial_start_time = dict['25']
-        self.timetrial_subsessionid = ['1']
-        self.trackid = dict['5']
+        self.timetrial_subsession_id = ['1']
```

Comment:
`time_trial_subsession_id` maybe? or maybe even just `tt_subsession_id`? but it might be confusing if people dont know what tt is

Reply:
ttrating is used on driver career profiles, so I think tt would be safe to use